tests/integration/product-type-export.spec.js

```diff
@@ -205,7 +205,7 @@ const mockAttributes = mockProductTypes.reduce((attributes, type) =>
 
 let OUTPUT_FOLDER
 
-const before = function setup () {
+const before = function setup (keys) {
```

Comment:
I don't really understand what this is supposed to do

Reply:
This setup function removes and then creates several productTypes with automatically generated keys. In my test, I wanted to test also a use case when the key is not set so I added this `keys` parameter and when creating productType I check if we have a key for this Nth productType and if it is null I delete that generated key.